exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/mergereceiver/MergingRecordBatch.java

```diff
@@ -84,6 +64,26 @@
 import com.sun.codemodel.JExpr;
 
 /**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you move this to the top of the file? It should actually come before the package statement.